File: src/test/java/com/evanhiggins/farmruns/FarmRunsPluginTest.java
package com.evanhiggins.farmruns;

import net.runelite.client.RuneLite;
import net.runelite.client.externalplugins.ExternalPluginManager;

public class FarmRunsPluginTest
{
	public static void main(String[] args) throws Exception
	{
		ExternalPluginManager.loadBuiltin(FarmRunsPlugin.class);
		RuneLite.main(args);
	}
}

File: src/test/java/com/evanhiggins/farmruns/FarmRunsPluginTest.kt
package com.evanhiggins.farmruns

import net.runelite.client.RuneLite
import net.runelite.client.externalplugins.ExternalPluginManager

object FarmRunsPluginTest {
    @Throws(Exception::class)
    @JvmStatic
    fun main(args: Array<String>) {
        ExternalPluginManager.loadBuiltin(FarmRunsPlugin::class.java)
        RuneLite.main(args)
    }
}